app/services/the_movie_db.rb

```diff
@@ -0,0 +1,100 @@
+class TheMovieDb
+  LIMIT = 650
+  AGE_ADULT = 18
+  MIN_AGE = 0
+  MAX_AGE = 17
+
+  def call
+    genres_request
+    companies_request
+    people_request
+    movies_request
+    actors_request
+    movie_staffs_request('director')
+    movie_staffs_request('producer')
+    movie_staffs_request('editor')
+    movie_staffs_request('compositor')
+    movie_staffs_request('artist')
+    movie_staffs_request('screencaster')
+  end
+
+  private
+
+  def genres_request
+    genres = Tmdb::Genre.list
+    Genre.create(genres['genres'])
+  end
+
+  def companies_request
+    (1..LIMIT).each do |i|
+      result = Tmdb::Company.detail(i)
+      company = { name: result['name'] }
+      Company.create(company)
+    end
+  end
+
+  def people_request
+    (1..LIMIT).each do |i|
+      result = Tmdb::Person.detail(i)
+      country = Country.find_or_create_by(name: result['place_of_birth']&.split&.last)
+      country&.save
+      person = { name: result['name']&.split&.first, surname: result['name']&.split&.last,
+                 birthday: result['birthday'], deathday: result['deathday'], biography: result['biography'],
+                 country_id: country[:id],
+                 married: Faker::Boolean.boolean }
+      Person.create(person).save
```

Comment:
create and save?

ar = AR.new(name: params[:name])
ar.errors unless ar.save


read about using **save, new, create, save!, update** and else